tsconfig.json

```diff
@@ -16,24 +16,7 @@
     "noEmit": true,
     "noFallthroughCasesInSwitch": true,
     "jsx": "react-jsx",
-    "types": ["vite/client", "vite-plugin-svgr/client"],
-    "paths": {
```

Comment:
why are we taking this out?

Reply:
i didn't take the types out just the path

Reply:
okay